### ISSUE TITLE
Hive Regal Slavemaker will seriously mess you up

### ISSUE THREAD

**Issue by Reloque** [HellscreamWoW/Tracker#878]
Not sure how to explain it, so I'll just do a narrative. I engaged a [Hive Regal Slavemaker](http://www.wowhead.com/npc=11733)s(11733) in Hive Regal, Silithus. Its spell, [Poison Mind](http://www.wowhead.com/spell=19469)(19469) causes you to lose control of your character and run (into the air on uneven ground...?) out of the Slavemaker's aggro range and back, thus restarting the encounter and allowing for an infinite loop if not for the occasional one to two seconds from the start of the encounter to when it casts Poison Mind.

Being a rogue, I used these two seconds to pull a Kidney Shot then finish it off. Unfortunately I soon realized that I was still tethered to the spot where the Slavemaker's corpse was: walking too far away caused me to again lose control of my character as he ran back to spot where the Slavemaker had despawned.

At this point I was going to relog, but rather than wait twenty seconds for that I used my cloak to teleport to Stormwind. As soon as the map loaded, my character was mindlessly pulled away, through the walls of the portal room (which is under the city, I learned) and then under the map. I exited the game immediately and did the standard fall-to-your-death + exit game before spirit release -> revived at graveyard.

In summary, that Qiraji Slavemaker mind-controlled me from beyond the grave on another continent, and only managed to kill me by forcing my body through the walls at Stormwind.
